spec/javascript/e2e/LeaseUpApplicationsPage/status_update.e2e.js

```diff
@@ -0,0 +1,119 @@
+import {
+  DEFAULT_E2E_TIME_OUT,
+  FIRST_ROW_LEASE_UP_APP_ID,
+  LEASE_UP_LISTING_ID,
+  SECOND_ROW_LEASE_UP_APP_ID,
+  THIRD_ROW_LEASE_UP_APP_ID
+} from '../../support/puppeteer/consts'
+import {
+  checkForStatusUpdateSuccess,
+  selectStatusDropdownValue,
+  fillOutAndSubmitStatusModal,
+  unselectedStatusMenuItem
+} from '../../support/puppeteer/steps/leaseUpStatusSteps'
+import sharedSteps from '../../support/puppeteer/steps/sharedSteps'
+import SetupBrowserAndPage from '../../utils/SetupBrowserAndPage'
+let testBrowser
+
+const firstRowStatusDropdown = '.rt-tr-group:first-child .rt-td .dropdown .dropdown-button'
+
+const waitForLeaseUpAppTableToLoad = async (page) => {
+  await sharedSteps.goto(page, `/lease-ups/listings/${LEASE_UP_LISTING_ID}`)
+  await page.waitForSelector('#root')
+  await page.waitForSelector(firstRowStatusDropdown)
+}
+
+describe('LeaseUpApplicationsPage status update', () => {
+  describe('using the individual row status dropdown', () => {
+    test(
```

Comment:
this was copied over from spec/javascript/e2e/LeaseUpPage.e2e.js then refactored significantly